src/Control/Lens/Internal.hs

```diff
@@ -41,7 +41,3 @@ import Control.Lens.Internal.Prism
 import Control.Lens.Internal.Review
 import Control.Lens.Internal.Setter
 import Control.Lens.Internal.Zoom
-
-#ifdef HLINT
```

Comment:
Now that this is gone, we should be able to get rid of the `{-# LANGUAGE CPP #-}` line above.